app/src/main/java/com/example/lhi06/residue/SaveReviews.java

```diff
@@ -0,0 +1,72 @@
+package com.example.lhi06.residue;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.widget.Toast;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SaveReviews {
+
+    private final static String FILENAME = "albumReviews.json";
+    private final static Gson gson = new Gson();
+    private final static Type type = new TypeToken<List<AlbumReview>>() {}.getType();
+    private static SaveReviews instance = null;
+
+    public static SaveReviews getInstance() {
+        if (instance == null) {
+            instance = new SaveReviews();
+        }
+        return instance;
+    }
+
+    public static String toJsonString(@NonNull final List<AlbumReview> reviews) {
+        return gson.toJson(reviews, type);
+    }
+
+    public static List<AlbumReview> toListOfReviews(final String jsonString) {
```

Comment:
it might be null if someone new uses the app and the app tries to load something but there isnt anything there, should i still put it as null or?